pkg/sql/catalog/tabledesc/validate_test.go

```diff
@@ -175,8 +175,9 @@ var validationMap = []struct {
 			"DefaultExpr": {
 				status: todoIAmKnowinglyAddingTechDebt,
 				reason: "initial import: TODO(features): add validation"},
-			"Hidden":       {status: iSolemnlySwearThisFieldIsValidated},
-			"Inaccessible": {status: iSolemnlySwearThisFieldIsValidated},
+			"GeneratedAsIdType": {status: iSolemnlySwearThisFieldIsValidated},
```

Comment:
was the field actually validated? :) 

you should add something here: https://github.com/cockroachdb/cockroach/blob/6fafc569ad64c70654345dd6c685c6b32991307b/pkg/sql/catalog/tabledesc/validate.go#L685

it should make sure the descriptor is not an an incoherent state. e.g., a `computed` column cannot be a `generatedAsIdentity` column.

Reply:
@rafiss  Done, check [here](https://github.com/cockroachdb/cockroach/pull/68324/files#diff-3feb8ec23c24149443848d5a36e11d9dd93cfddf4a95363a80c4b79605664667R706-R709). Also added [a `NOT NULL` constraint](https://github.com/cockroachdb/cockroach/pull/68324/files#diff-01f73e0f6ec31b2bbaef6117ca07bc2a1870550909a5ec2bfe8b011387ad4f67R544-R546), and corresponding [parser tests](https://github.com/cockroachdb/cockroach/pull/68324/files#diff-dccba38be048680977583650bddede7895ab54cccce0fbccabcd66f2044402e8R2276-R2302) and [logic tests](https://github.com/cockroachdb/cockroach/pull/68324/files#diff-fa7c476e990651baf64186090334103fb4acfd5ff9b37e6f699be44e27d61e61R493).